seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/BasicType.java

```diff
@@ -24,6 +24,7 @@ public class BasicType<T> implements SeaTunnelDataType<T> {
 
     public static final BasicType<String> STRING_TYPE =
             new BasicType<>(String.class, SqlType.STRING);
+    public static final BasicType<String> JSON_TYPE = new BasicType<>(String.class, SqlType.JSON);
```

Comment:
We already has https://github.com/apache/seatunnel/blob/dev/seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/CommonOptions.java#L33. So I think we can reuse this option. Not create new type.